### PR TITLE
Feature/scrollable containers

### PR DESCRIPTION
**Pull request by r1g023**
# Description
- The left sidebar and main body should scroll independently.

Issue # 
- issue #35 (let me know if i need to update this)

Describe what you're adding
- Added scrollable feature to both sidebar and main-card containers

# QA

List out how to test your changes and add a screenshot of what it should look like, when applicable.
- you can test changes by checking out the 'feature/scrollable-containers' branch
- see screenshot attached 
- scrollableContainers

### CODE REVIEW

**Review by zeeberry — COMMENTED** [review]
Looks good. Let's remove the console log and it should be good to go. I left another comment to reduce redundancy, but that can be a follow up task.

File: src/components/PaletteColors.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+function PaletteColors() {
+  return (
+    <Box
+      flex="5"
+      display="flex"
+      justifyContent={"flex-start"}
+      alignItems={"center"}
+      color={"#000000"}
+    >
+      <IconButton
```
[the remark this answers is not in the document]
Comment (r1g023) [reply src/components/PaletteColors.js]: Thank you for the feedback, Z. I have refactored the PaletteColors component according to your suggestion. I forgot that I could loop through only one icon to get all eight chipColor icons. This has been implemented. Please let me know if it looks fine under the existing branch on issue #39.

File: src/components/PaletteCopyButton.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Button } from "@chakra-ui/react";
+import { CopyIcon } from "@chakra-ui/icons";
+
+async function copyToClipboard() {
+  const headerHexText = document.querySelector(".headerHexText").textContent;
+
+  try {
+    await navigator.clipboard.writeText(headerHexText);
+    console.log("Copied Hex text, test");
```
[the remark this answers is not in the document]
Comment (r1g023) [reply src/components/PaletteCopyButton.js]: @zeeberry I wanted to let you know that I have removed the console log from the PaletteCopyButton. Additionally, I removed another console log from the same copy hex function that I had forgotten to remove earlier. While I was working on it, I decided to create a new section within the components folder. I added the main, navigation, and sidebar folders within this new section to separate the code and make it much more readable. Thank you for letting me do this - it's much easier on my eyes now!

**Comment by r1g023** [comment]
@zeeberry I recall that you suggested I use React (and Material UI which I noticed has the clipboard option) to implement the copy/paste functionality instead of using plain JavaScript. If you have the time, could you please add the next task for me to implement this feature? Let me know if you have any questions. Thank you again for your feedback.

**Review by zeeberry — APPROVED** [review]
nice updates 👍🏼

**Pull request merged**